introduktion/sammanfattning.md

```diff
@@ -3,21 +3,24 @@
+## Nuläge
+Sverige saknar idag en nationell samsyn kring hur identitets- och behörighetshantering ska fungera i en digital samverkanskontext som sträcker sig över verksamhetsdomäner och organisationsgränser. De nuvarande lösningarna bygger till stor del på bilaterala avtal eller olika former av sammanslutningar för federerad åtkomst, ofta begränsade till specifika verksamhetsdomäner eller användningsområden – så kallade federationer.
 
-För att federationer ska verka genom en sammanhållen nationell samsyn med avseende på hur identitets- och behörighetshantering ska fungera över organisationsgränser krävs att parter kan komma överens över verksamhetsgränser och kunskapsdomäner om formerna för vilka aktörer som får ingå i en sammanslutning, vilka standarder som ska följas, vilka säkerhetslösningar som ska nyttjas, vilka administrativa processer som ska följas och hur kostnader ska fördelas. Genom avtal dokumenterar parterna att man är överens om att följa det federativa systemets regler och utifrån dessa lita på information som andra organisationer förmedlar om sina användare. Ibland förekommer också någon form av granskning och det kan finnas möjlighet att begära skadestånd om en skada uppstår på grund av att dessa regler ej följts.
+Offentliga e-tjänster i Sverige har traditionellt utformats inom ramen för respektive organisations ansvarsområden. Detta har bidragit till fragmentering och en stor variation av e-tjänster med skiftande kvalitet. Utvecklingen har också fått motta kritik, eftersom utformningen ofta baseras på den egna myndighetens vinster och effektivisering, snarare än att fokusera på medborgarens behov eller de samhälleliga vinsterna i stort. 
 
-Tillit mellan parter och deras digitala tjänster etableras vanligtvis genom något någon form av *[tillitsmodell](tillit.md)*. Den tekniska reliseringen utgår från en uppsättning av tekniska protokoll och specifikationer för att möjliggöra federerad åtkomst.
+De [federationer som existerar idag](federationer.md#dagensfederationer) har främst fokuserat på att skapa digitala tjänster för interaktion mellan individ och verksamhet, med government to citizen (G2C) som den primära leveransmodellen. Även om dessa federationer sektorsvis utgör effektiva och säkra infrastrukturer för förmedling av gränsöverskridande identitets- och behörighetsintyg för åtkomst till e-tjänster, saknas en samordnad nationell strategi för att stödja den svenska välfärdsmodellen fullt ut.
+De befintliga lösningarna, som konceptuellt verkar kunna fungera ihop, har i praktiken visat sig vara svåra att skala upp och integrera – något som också hämmar digitaliseringstakten. Dagens fragmenterade landskap behöver därför utökas med government to government (G2G) leveransmodeller som kan dels ta vara på redan gjorda investeringar, dels stärka stödet digital samverkan mellan verksamheter och över sektorsgränser. 
 
+## Färdplan
```

Comment:
Vad tror du om att ta med en kort sammanfattning av målbild?
## Målbild:
Den föreslagna infrastrukturmodellen främjar en decentraliserad hantering av tillitsinformation för digitala tjänster. Modellen möjliggör ett delat ansvar för tillit och stödjer användning av olika tekniska protokoll. Den är flexibel och anpassad för att hantera gränsöverskridande identitets- och åtkomsthantering för både användare och system.

Reply:
Bra idé! Det gör vi! ...användare och systemanvändare ;-)
Vi kan väl uppmana arbetsgruppen också titta igenom så gör jag en revidering utifrån alla kommentarer som kommer in. Låter det ok?